### PR TITLE
Clean up experiment flags.

### LINKED ISSUE #6254: Clean up experiment flag for visibility-v2

### PR DESCRIPTION
**Pull request by lannka**
Closes #6254 #6156

### CODE REVIEW

File: build-system/global-configs/prod-config.json [hunk 1]
```diff
@@ -1,26 +1,21 @@
...
   "amp-scrollable-carousel": 1,
-  "amp-inabox": 1,
-  "amp-ad-loading-ux": 1,
   "ios-embed-wrapper": 1,
   "amp-apester-media": 1,
   "amp-accordion-session-state-optout": 1,
   "amp-playbuzz": 1,
   "make-body-relative": 1,
   "chunked-amp": 1,
-  "visibility-v2": 1,
```
Comment (zhouyx) [inline build-system/global-configs/prod-config.json]: let's also remove it from experiments.js.

**Comment by jridgewell** [comment]
Want to prune the SW's blacklist too?

**Comment by lannka** [comment]
> Want to prune the SW's blacklist too?

@jridgewell what's that?

File: build-system/global-configs/prod-config.json (hunk 1 above)
Comment (lannka) [inline build-system/global-configs/prod-config.json]: good catch

**Comment by lannka** [comment]
the new presubmit doesn't allow changing other files in the same PR. will do in a separate PR.

**Pull request merged**